docs/ReleaseNotes.md

```diff
@@ -1,4 +1,9 @@
 
+# UADK Release v2.6 Dec 2023
+
+## Features:
+- Support
```

Comment:
- Support sm4-xts GB
 - Support queue depth configurable feature
 - Support user-defined custom data structure function for uadk v1
 - Support aes-cts algorithm
 - Support xts mode DIF function for uadk v1